src/app/core/entity-details/related-entities/related-entities.component.ts

```diff
@@ -112,12 +112,13 @@ export class RelatedEntitiesComponent<E extends Entity> implements OnInit {
     this.data = await this.getData();
     this.filter = this.initFilter();
 
-    if (this.showInactive === undefined) {
-      // show all related docs when visiting an archived entity
-      this.showInactive = this.entity.anonymized;
-    }
-
```

Comment:
did you intentionally remove this logic?

Reply:
> did you intentionally remove this logic?

Ah, thanks for noticing this; let me add this.